common/locales/en/messages.json

```diff
@@ -14,6 +15,7 @@
   "messageEquipped": " <%= itemText %> equipped.",
   "messageUnEquipped": "<%= itemText %> un-equipped.",
   "messageMissingEggPotion": "You're missing either that egg or that potion",
+  "messageInvalidEggPotionCombo": "Invalid egg+potion combination. Try hatching a different egg!",
```

Comment:
Probably so. Maybe "You can't hatch that egg with this potion. Try a different egg!"?